.github/workflows/ci.yml

```diff
@@ -13,4 +13,4 @@ jobs:
         java-version: 8
         cache: sbt
     - uses: sbt/setup-sbt@v1
-    - run: sbt +test
+    - run: sbt scalafmtSbtCheck +scalafmtCheckAll +test
```

Comment:
The trend seems to be splitting this up into a new action? - https://github.com/sbt/librarymanagement/blob/cc3d911e0b3a573078b77ce856b411d82dd070f2/.github/workflows/format.yml